lib/libyui_client/widgets/combobox.rb

```diff
@@ -4,39 +4,8 @@ module LibyuiClient
   module Widgets
     # Class representing a ComboBox in the UI. It can be YComboBox.
     class Combobox < Widgets::Base
-      # Sends action to select the item in combobox.
-      # @param item [String] item to select in combobox.
-      # List of items can be retrieved from JSON "items"->"label" manually or by using 'combobox(filter).items'.
-      # @return [Combobox] in case action is successful
-      # @raise LibyuiClient::Error::ItemNotFoundInWidgetError in case value is not found in combobox.
-      # @example Select "Force NFSv3" item in combobox with id "nfs_version"
-      #   {
-      #     "class": "YComboBox",
-      #     "debug_label": "NFS Version",
-      #     "icon_base_path": "",
-      #     "id": "nfs_version",
-      #     "items": [
-      #       {
-      #         "label": "Any (Highest Available)",
-      #         "selected": true
-      #       },
-      #       {
-      #         "label": "Force NFSv3"
-      #       }
-      #     ],
-      #     "items_count": 5,
-      #     "label": "NFS &Version",
-      #     "value": "Any (Highest Available)"
-      #   }
-      # @example
-      #   app.combobox(id: 'nfs_version').select('Force NFSv3')
-      def select(item)
-        action(action: Actions::SELECT, value: item)
-        self
-      end
-
       # Returns the list of items available to select from combobox.
-      # @return [Array] array of Strings.
+      # @return [Array] array of String objects
```

Comment:
As far as I know, everything is Object in Ruby. Not sure this comment makes the description more correct. But you can leave it if you like it more.

Reply:
I have seen more frequently used 'array of strings' instead of pluralize String, in fact checking yard cheatsheet, seems that `@return [Array<String>] array of strings` could be a good choice. Good that you noticed.